pyproject.toml

```diff
@@ -0,0 +1,71 @@
+[build-system]
+requires = ["flit_core >=3.5,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "quantecon"
+authors = [{name = "QuantEcon Project", email = "admin@quantecon.org"}]
+classifiers = [
+    'Development Status :: 4 - Beta',
+    'Operating System :: OS Independent',
+    'Intended Audience :: Science/Research',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.5',
+    'Topic :: Scientific/Engineering',
+]
+keywords = [
+    'quantitative', 
+    'economics'
+]
+dynamic = ["description", "version"]
+requires-python = ">=3.7"
+dependencies = [
+    'numba',
+    'numpy',
+    'requests',
+    'scipy>=1.0.0',
+    'sympy',
+]
+
+[project.optional-dependencies]
+testing = [
+    "pytest",
+    "coverage",
+    "flake8",
+    "numpy",
+    "scipy",
+    "pandas",
+    "numba",
+    "sympy",
+    "ipython",
```

Comment:
`ipython` is used only when building the documentation, but not used in testing.